### PR TITLE
checkbox fix for registration, focus first input

### PR DESCRIPTION
**Pull request by jwong-ping**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -225,6 +225,11 @@ export default class AuthnWidget {
       });
     }
 
+    let firstTextInput = document.querySelector("input[type=text]:not(:disabled)");
```
Comment (mketchum) [inline src/index.js:228]: Do you think we should include date, email, and phone as field types to focus as well?

Comment (mketchum) [reply src/index.js:228]: We might want to clean up all the `autofocus` references in the templates where it makes sense.

Comment (jwong-ping) [reply src/index.js:228]: hmm, I didn't notice autofocus was placed in some of the other partials already, perhaps it's better to place this in a post render callback for just registration instead to keep that level of control.

Comment (mketchum) [reply src/index.js]: @khosravian What do you think about this? Should we have global autofocus or leave it up to the templates to determine which field gets the focus?

Comment (khosravian) [reply src/index.js]: @mketchum I think we need both, global autofocus on the first field and a way to override the behavior at the template level.

Comment (jwong-ping) [reply src/index.js]: global autofocus first input (date, email, phone, pass, text) unless autofocus specified in template

**Review by khosravian — DISMISSED** [review]

File: src/index.js [hunk 2]
```diff
@@ -793,6 +791,21 @@ export default class AuthnWidget {
     if (this.store.state.showCaptcha && this.grecaptcha) {
       this.grecaptcha.render(this.captchaDivId);
     }
+
+    let alreadyAutofocus = false;
+    let inputElements = document.querySelectorAll("input:not(:disabled)")
+    inputElements.forEach(ele => {
```
Comment (mketchum) [inline src/index.js:797]: NIT: I think we can query on attributes as well and avoid the for loop. Something like `'input:not(:disabled)[autofocus]'` should allow for it.

**Review by mketchum — APPROVED** [review]

**Review by khosravian — APPROVED** [review]

**Pull request merged**